prac_09/silver_service_taxi.py

```diff
@@ -0,0 +1,24 @@
+"""
+Silver Service Taxi Class
+child of taxi class
+"""
+from taxi import Taxi
+
+
+class SilverServiceTaxi(Taxi):
+    """ A child of the taxi class, therefore inherits from taxi and car classes """
+    flagfall = 4.50
+
+    def __init__(self, name, fuel, fanciness: float):
```

Comment:
In this __init__ method it is simpler to use **kwargs (key word arguments)

    def __init__(self, fanciness, **kwargs):
        """Initialise a specialised taxi, based on parent class Taxi."""
        super().__init__(**kwargs)